.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "MG5aMC/mg5amcnlo"]
 	path = MG5aMC/mg5amcnlo
 	url = https://github.com/mg5amcnlo/mg5amcnlo
-	branch = gpucpp
+	branch = gpucpp_wrap
```

Comment:
Ok this I need to fix before going to master.
NOTE: I am not using gpucpp_wrap anyway, see #886